### ISSUE TITLE
Text input border color with low contrast does not fail WCAG 1.4.11: Non-text Contrast

### ISSUE THREAD

**Issue by BogdanCerovac** [dequelabs/axe-core#3697]
### Product

axe-core

### Product Version

4.4.2

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
Input type text with border of 1px and color #f9c66b on white background should fail WCAG 1.4.11: Non-text Contrast (Contrast against white is 1.57:1, so way under 3:1)

#### Actual
Used axe extension and it didn't catch the problem.

#### How to Reproduce

Please visit https://jsfiddle.net/pa3xk714/show and "Run this fiddle", then run axe extension and check results.

It catches another contrast problem but not this one.

**Comment by straker** [comment]
Thanks for the issue. Axe-core color-contrast currently does not apply to WCAG 1.4.11, only WCAG 1.4.3. As far as I'm aware we don't have any plans to cover more than just 1.4.3, though there has been some discussion about testing [font icons for 3:1 contrast](https://github.com/dequelabs/axe-core/issues/1826).

**Comment by BogdanCerovac** [comment]
Aha, OK, thank you.
